extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/static_resource/StaticResourceBuildTimeConfig.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.vertx.http.runtime.static_resource;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "static-resource", phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+public class StaticResourceBuildTimeConfig {
+    /**
+     * The endpoint from which the static resources will be served.
+     */
+    @ConfigItem(name = ConfigItem.PARENT)
+    public String endpoint;
+
+    /**
+     * The default landing page of static resource.
+     */
+    @ConfigItem(defaultValue = "index.html")
+    public String index;
```

Comment:
Optional?

Reply:
Yes, I hadn't rebased with the new config PR.